static/js/ktl-component/teach/icons/anchor.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <circle cx="20" cy="20" r="20" fill="#F5F5F5"/>
+    <mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="9" y="12" width="22" height="17">
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M22.5815 13.7015C24.3931 11.8908 27.3294 11.8908 29.141 13.7015C30.9527 15.5133 30.9527 18.4507 29.141 20.2625L24.539 24.8645L23.093 23.4185L27.695 18.815C28.6872 17.7988 28.6774 16.1736 27.6732 15.1693C26.6689 14.1651 25.0437 14.1553 24.0275 15.1475L19.4255 19.751L17.9795 18.305L22.5815 13.7015ZM12.3238 25.4052C13.3281 26.4094 14.9533 26.4192 15.9695 25.427L20.5715 20.825L22.019 22.271L17.4155 26.8745C15.6037 28.6859 12.6666 28.6855 10.8552 26.8738C9.04389 25.062 9.04423 22.1249 10.856 20.3135L15.458 15.7115L16.904 17.1575L12.302 21.7595C11.3098 22.7757 11.3196 24.4009 12.3238 25.4052Z" fill="white"/>
+    </mask>
+    <g mask="url(#mask0)">
+        <rect x="8" y="8" width="24" height="24" fill="#27282C" fill-opacity="0.7"/>
+    </g>
+</svg>
```

Comment:
Let's reuse the [existing one](https://github.com/JetBrains/kotlin-web-site/blob/04b36d7bcb3c742471d10a4196235aaaa2e6b8d1/static/js/page/index/images/anchor.svg).
I suggest moving it to some shared folder.

Reply:
Or we can do it differently. A CSS class that adds the icon to a header would be better.